### ISSUE TITLE
5 things you can do to make your site more accessible

### ISSUE THREAD

**Issue by diego-codes** [diego-codes/blog#3]
Accessibility does not have to be difficult to tackle in your project. The (relatively) simple things you can do to make your website accessible:

- Turn on outlines
  - You can style them to match your brand
- Identify your main content area and have a "Skip to Content" button for it
- Make hidden focusable elements unfocusable
- Check your site for color contrast
- Dim your screen and use a screen reader to complete your tasks
- All of your forms inputs have a label and correct type
